### DISCUSSION TITLE
OpenApi support

### DISCUSSION THREAD

**Discussion by Cjewett** [jsonyte/jsonyte#85] [Q&A] [answered]
Hello @robertcoltheart,

I had a question regarding OpenApi support. When testing it out I noticed that the OpenApi/Swagger UI showed the response as normal JSON. After making an API call it would return the JSON in JSON:API format though. Is this library expected to work with OpenApi out of the box?

Thanks!

**Comment by robertcoltheart** [comment, marked as answer]
You'll need to set your content-type and accept headers manually for Swashbuckle/Swagger, such that you can select the request/response type in a drop-down using Swagger. That should let you send/receive responses as JSON:API. However the default model JSON that Swagger will show for POST entries won't show up as JSON:API formatted. 

I started putting together a set of ASP.NET Core integrations, including Swagger, over at https://github.com/jsonyte/jsonyte-aspnetcore, but I'm not really using JSON:API anymore, so don't expect anything in the short/medium term.

**Comment by Cjewett** [comment]
Thanks, appreciate the response! Good to know what the current state is and I'll relay that information to folks so they can make decisions off of it.
